### PR TITLE
JavaBasic done

### PR DESCRIPTION
**Pull request by rdankhara**
Topics covered 
1.

### CODE REVIEW

File: src/JavaBasic.java [hunk 1]
```diff
@@ -1,5 +1,103 @@
...
+    public static void removespace() { // method for removing space in string
+        String sample = "This is very easy programme"; // sample string
+        System.out.print( "3) String without space will look like = " ); // result description for printing string
+        System.out.println( sample.replaceAll( "\\s", "" ) ); // print result string without space
+    }
+
+    public static void extractspechar() { // method for extracting special characters from string
+        String sample = "This $300 is equivalent of £200 but not for @100"; // sample string with special characters
+        System.out.print( "4) Removing special Characters = " ); // result description for printing string
+        System.out.println( sample.replaceAll( ("[^a-zA-Z0-9]"), " " ) );
```
Comment (rdankhara) [inline src/JavaBasic.java:24]:
```suggestion
        System.out.println( sample.replaceAll( ("[^a-zA-Z0-9]"), "" /* no space */ ) );
```
with your code you are replacing special characters by space, but you need to remove it so replace by empty string ""  , good that you used regex

File: src/JavaBasic.java [hunk 2]
```diff
@@ -1,5 +1,103 @@
...
+    } //
+
+    public static void extractnumber() { // method for extracting number from string
+        String sample = "The cost of iPhone is £1,250"; // sample string to extract numbers
+        System.out.print( "5) The cost of iPhone is " ); // result description for printing string
+        //printing result for extracting number from string
+        System.out.println( sample.replaceAll( "[^0-9]", "" ) );
+    }
+
+    public static void inttonum() { // method for integer to bouble
```
Comment (rdankhara) [inline src/JavaBasic.java:34]: good naming is intToNum (camel case)

**Review by rdankhara — COMMENTED** [review]
over all ok, one place you need to use "" instead " ", 
method names should be starting with lower case but new word should be capital, its called camel casing
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841 

so your method names 
remove - ok as only one word
findlength() - should findLength() 
removespace() - should be removeSpace() 
extractspechar() - should be extractSpecialChar() and so on, 

it will be good if you apply these changes, commit and push branch again.

File: src/JavaBasic.java (hunk 1 above)
Comment (pbksoft2019) [reply src/JavaBasic.java:24]: i have changed From " " to ""
and also changed method names as you suggested with camel case. i will create push it again now

**Pull request merged**